### PR TITLE
Add `--key-config-path` alias 

### LINKED ISSUE #185: Add `--key-config-path` as preferred alias to `--key-store-path`
See: https://github.com/Consensys/web3signer/pull/826

### PR DESCRIPTION
**Pull request by bgravenorst**
Add `--key-config-path` as a preferred alias for  `key-store-path`.
Update ConsenSys -> Consensys

fixes #185

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const redocusaurus = [
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: "ConsenSys Web3Signer",
+  title: "Consensys Web3Signer",
```
Comment (siladu) [inline docusaurus.config.js:43]: Not sure which way is correct but maybe this should be Web3signer too

Comment (bgravenorst) [reply docusaurus.config.js:43]: Maybe we should do Web3Signer -> Web3signer in a separate PR? We'll need to ensure the codebase gets updated as well, see: https://github.com/Consensys/web3signer/blob/master/CONTRIBUTING.md

Comment (siladu) [reply docusaurus.config.js:43]: I don't know which is better/more correct. If there's a requirement to change it though then agree separate PR is best

Comment (bgravenorst) [reply docusaurus.config.js:43]: Will confirm which is the preferred name, and update it (if required) in a seperate PR.

File: docs/Reference/CLI/options.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ data-path: "/Users/me/my_node/data"
 
 Directory in which to store temporary files.
 
-### `key-store-path`
+### `key-store-path`, `key-config-path`
```
Comment (siladu) [inline docs/Reference/CLI/options.md:86]: key-config-path is the preferred name so maybe this should come first?

Comment (siladu) [reply docs/Reference/CLI/options.md:86]: I'm not sure updating the title is the right thing either. Compare to other options with an alias... https://doc-web3signer-git-fork-bgravenorst-web-185-infura-web.vercel.app/Reference/CLI/options#logging

File: docusaurus.config.js (hunk 1 above)
Comment (siladu) [reply docusaurus.config.js:43]: @bgravenorst just submitted more comments

File: docs/Reference/CLI/options.md (hunk 2 above)
Comment (bgravenorst) [reply docs/Reference/CLI/options.md:86]: I'll make `key-config-path` first. We do the aliases for `--logging | -l`, `--version | -V`, and '--help | -h` differently due to the single dash (-) used by the alias. 

But generally we update the title to display the aliases. For example, in [Teku](https://docs.teku.consensys.net/reference/cli#data-base-path-data-path).

Comment (bgravenorst) [reply docs/Reference/CLI/options.md:86]: On a seperate note, shouldn't we also create an alias for `--key-store-config-file-max-size`?

Comment (siladu) [reply docs/Reference/CLI/options.md]: Thanks @bgravenorst that makes sense. Yes, maybe we should create that alias too, good spot, I'll bring it up with the team

**Review by siladu — APPROVED** [review]
LGTM

**Pull request merged**